core/blockchain.go

```diff
@@ -1925,8 +1928,12 @@ func (bc *BlockChain) writeBlockAndSetHead(block *types.Block, receipts []*types
 	if err != nil {
 		return NonStatTy, err
 	}
-	if reorg && mux != nil {
-		mux.Post(NewSealedBlockEvent{Block: block})
+	if reorg {
+		bc.highestVerifiedBlock.Store(types.CopyHeader(block.Header()))
+		bc.highestVerifiedBlockFeed.Send(HighestVerifiedBlockEvent{Header: block.Header()})
```

Comment:
would it get stuck here, if the VoteManage did not consume the channel in time? As it is a buffered channel with size of `highestVerifiedBlockChanSize`

Reply:
won't stuck.
when do sync, will imports many blocks quickly,
![image](https://github.com/user-attachments/assets/1df32bee-f67d-46ca-bfe4-28bed0115e98)
startVote is false, so VoteManage can handle the HighestVerifiedBlockEvent quickly, won't block then.

Reply:
when if vote is not enabled? 
https://github.com/bnb-chain/bsc/blob/master/eth/backend.go#L358

Reply:
> when if vote is not enabled? https://github.com/bnb-chain/bsc/blob/master/eth/backend.go#L358

so no channel subscrible the event, highestVerifiedBlockFeed.Send will return directly